test/util.jl

```diff
@@ -1,9 +1,13 @@
 import Mocking: ignore_stderr, to_array_type
 
-# TODO: Not a great test...
```

Comment:
You can leave this comment in if you want. The code is at fault for the strangeness of this test